packages/manager/modules/dbaas-logs/src/logs/detail/encryption-keys/home/home.component.js

```diff
@@ -0,0 +1,14 @@
+import controller from './logs-encryption-keys-home.controller';
+import template from './logs-encryption-keys-home.html';
+import './home.scss';
+
+export default {
+  controller,
+  controllerAs: '$ctrl',
```

Comment:
**nitpick:** No need to specify `controllerAs` while using default.
```suggestion
```

Reply:
Removed